src/dashboard/src/pages/Cluster/Pods.tsx

```diff
@@ -117,7 +118,7 @@ const Pods: FunctionComponent<Props> = ({ data: { config, workers }, query }) =>
     render: ({ memory }) => <>{humanBytes(memory)}</>,
     width: 'auto'
   } as Column<any>, {
-    title: 'GPU Utilization',
+    title: 'Available GPU Utilization',
```

Comment:
I think Jing said "Assigned" instead of "Available"?

Reply:
done